whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/Remark.java

```diff
@@ -16,8 +16,11 @@
 public class Remark implements Serializable {
     protected List<String> description;
 
-    public Remark(final List<String> description) {
+    protected List<String> remarks;
```

Comment:
Don't add remarks under remarks, that's not a valid RDAP response (see RFC 9083). Any "remarks:" attribute values need to be added as additional *description* elements.

Reply:
Done